cmd/standup/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine as builder
```

Comment:
Do we still need individual Dockerfiles? I thought everything was bundle together as the ECM docker image

Reply:
Fixed.